test/e2e/e2e.go

```diff
@@ -258,11 +258,64 @@ var _ = ginkgo.Describe("[efs-csi] EFS CSI", func() {
 				framework.Failf("timed out verifying exec in pod succeeded")
 			}
 		})
+
+		testEncryptInTransit := func(f *framework.Framework, encryptInTransit *bool) {
+			ginkgo.By("Creating efs pvc & pv")
+			volumeAttributes := map[string]string{}
+			if encryptInTransit != nil {
+				if *encryptInTransit {
+					volumeAttributes["encryptInTransit"] = "true"
+				} else {
+					volumeAttributes["encryptInTransit"] = "false"
+				}
+			}
+			pvc, pv, err := createEFSPVCPV(f.ClientSet, f.Namespace.Name, f.Namespace.Name, "/", volumeAttributes)
+			framework.ExpectNoError(err, "creating efs pvc & pv with no subpath")
+			defer func() { _ = f.ClientSet.CoreV1().PersistentVolumes().Delete(pv.Name, &metav1.DeleteOptions{}) }()
+
+			// If mount.efs is passed option tls, the mount table entry should be...
+			// 127.0.0.1:/ on /mnt/volume1 type nfs4 (rw,relatime,vers=4.1,rsize=1048576,wsize=1048576,namlen=255,hard,noresvport,proto=tcp,port=20052,timeo=600,retrans=2,sec=sys,clientaddr=127.0.0.1,local_lock=none,addr=127.0.0.1)
+			// Note the tls option is not actually there. The proof that tls is passed
+			// to mount.efs is the server is 127.0.0.1
+			// (stunnel proxy running on localhost)
+			// instead of the EFS DNS name
+			// (file-system-id.efs.aws-region.amazonaws.com)
+			command := "mount && mount | grep /mnt/volume1 | grep 127.0.0.1"
+			if encryptInTransit != nil {
+				if !*encryptInTransit {
+					command = fmt.Sprintf("mount && mount | grep /mnt/volume1 | grep %v", FileSystemId)
```

Comment:
What does the first `mount` do from `mount && mount` ?

Reply:
That is just to print it for debugging

Reply:
added msg